.github/workflows/quick-edit-changeset-for-workers-types-updates.yml

```diff
@@ -0,0 +1,29 @@
+name: "Quick Edit - Generate changesets for dependabot PRs"
+on:
+  pull_request_target:
+    paths:
+      - "packages/quick-edit-extension/package.json"
+
+jobs:
+  generate-changeset:
+    runs-on: ubuntu-latest
+    if: |
+      github.event.pull_request.user.login == 'dependabot[bot]'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 2
+          ref: ${{ github.head_ref }}
+          token: ${{ secrets.GH_ACCESS_TOKEN }}
+
+      - name: Install Dependencies
+        uses: ./.github/actions/install-dependencies
+
+      - name: Configure Git
+        run: |
+          git config --global user.email wrangler@cloudflare.com
+          git config --global user.name 'Wrangler automated PR updater'
+
+      - name: Generate quick-edit-extension changeset
+        run: node -r esbuild-register tools/dependabot/generate-dependabot-pr-changesets.ts ${{ github.event.number }} quick-edit-extension packages/quick-edit-extension/package.json dependabot-update-quick-edit-extension
```

Comment:
This will create a changeset for quick-edit-extension when dependabot bumps it's workers-types dependency